src/uct/tcp/tcp_ep.c

```diff
@@ -418,6 +418,49 @@ static inline void uct_tcp_ep_ctx_move(uct_tcp_ep_ctx_t *to_ctx,
     memset(from_ctx, 0, sizeof(*from_ctx));
 }
 
+static ucs_status_t uct_tcp_ep_setup_keepalive(uct_tcp_ep_t *ep)
+{
+#if UCT_TCP_EP_KEEPALIVE
+    uct_tcp_iface_t *iface = ucs_derived_of(ep->super.super.iface,
+                                            uct_tcp_iface_t);
+    int optval = 1;
```

Comment:
align by `=` with above
maybe make it `const`

Reply:
yep, fixed